src/CliCommand.php

```diff
@@ -17,39 +17,50 @@ class CliCommand extends \WP_CLI_Command {
   /**
    * Updates products sale discounted percentage.
    */
-  public function refreshSalePercentage(array $args, array $options) {
-    $options = wp_parse_args($options, ['all' => '']);
-    if ($options['all']) {
-      $product_ids = wc_get_products([
-        'return' => 'ids',
-        'limit' => -1,
-      ]);
-      static::updateSalePercentage($product_ids);
-      WP_CLI::success('Sale percentage have been sucessfully updated.');
+  public function refreshSalePercentage(array $args, array $assoc_args) {
```

Comment:
The original signature `array $args, array $options` was correct.  A shell command consists of arguments and options.

Reply:
On https://make.wordpress.org/cli/handbook/commands-cookbook/#annotating-with-phpdoc is  ` function hello( $args, $assoc_args ) {`.

I'd stick with what the offical documentation says.

Reply:
The documentation is wrong in this case. In addition, the original code used `$options` already.